File: tools/siclaw-sig/test-fixtures/java/Sample.java
package com.example.controller;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class PodController {
    private static final Logger logger = LoggerFactory.getLogger(PodController.class);

    public void reconcilePod(String podName, String namespace) {
        logger.info("Starting reconciliation for pod {}/{}", namespace, podName);
        logger.warn("Pod {} is in pending state for {} seconds", podName, 120);
        logger.error("Failed to create pod sandbox for {}: {}", podName, "timeout");
        logger.debug("Reconciliation step {} for pod {}", 1, podName);
    }

    public void healthCheck() {
        logger.info("Health check passed");
        logger.warn("Health check degraded: {}", "high latency");
    }

    public void startServer(int port) {
        logger.info("Server starting on port {}", port);
        logger.error("Server failed to bind to port {}: {}", port, "address in use");
        logger.trace("Detailed startup diagnostics for port {}", port);
    }
}

File: tools/siclaw-sig/test-fixtures/python/sample.py
import logging

logger = logging.getLogger(__name__)


def connect_to_database(host: str, port: int) -> None:
    logger.info("Connecting to database at %s:%d", host, port)
    logger.warning("Connection attempt %d of %d", 1, 3)
    logger.error("Failed to connect to %s: %s", host, "connection refused")


def process_request(request_id: str) -> None:
    logging.info("Processing request %s", request_id)
    logging.warning("Request %s took longer than expected", request_id)
    logging.error("Request %s failed: %s", request_id, "timeout")
    logging.debug("Request payload: %s", request_id)


def start_server(port: int) -> None:
    logger.info(f"Server starting on port {port}")
    logger.info("Server started successfully on port %d", port)
    logger.critical("Server failed to start: %s", "address in use")


def health_check() -> None:
    logging.info("Health check passed")
    logger.warning("Health check degraded: %s", "high latency")
